pyproject.toml

```diff
@@ -18,18 +18,35 @@ aiofiles = ">=22.1"
 obs-package-update = { git = "https://github.com/dcermak/obs-package-update", branch = "main" }
 GitPython = ">=3.1"
 
-[tool.poetry.dev-dependencies]
-black = ">=23.3, <24.1"
+[tool.poetry.scripts]
+package = "bci_build.package:main"
+scratch-build-bot = "staging.bot:main"
+
+[tool.poetry.group.dev.dependencies]
+ruff = ">=0.1.14"
 Sphinx = ">=4.4"
 pytest = ">=7.1"
 types-aiofiles = ">=22.1"
-reorder-python-imports = ">=3.8"
 pytest-asyncio = ">=0.20, != 0.22.0"
 pyyaml = ">=6.0"
 
-[tool.poetry.scripts]
-package = "bci_build.package:main"
-scratch-build-bot = "staging.bot:main"
+[tool.ruff.lint]
+select = [
+    # Pyflakes
+    "F",
+    # Pycodestyle
+    "E",
+    "W",
+    # isort
+    "I"
+]
+ignore = [
+    # too long lines…
+    "E501"
```

Comment:
why? we should set the line-length to the "right" value (whatever we consider right, BCI-tests uses 79)

Reply:
ok, I see we ignored this so far.